recipe/meta.yaml

```diff
@@ -13,18 +13,16 @@ source:
   sha256: {{ sha256 }}
 
 build:
+  noarch: python
   number: {{ build }}
   script: flit install --deps=none
-  skip: True  # [py2k]
 
 requirements:
   build:
-    - python
+    - python >=3
     - flit
-    - pytoml  # flit should depend on pytoml
-    - pip  # flit should depend on pip
```

Comment:
removed `pip` and `pytoml` from the build-time dependencies because they are now properly listed as dependencies of flit.